e2e/utils/tenv.ts

```diff
@@ -354,6 +396,20 @@ export function createTenv(cwd: string): Tenv {
         getStats() {
           throw new Error('Cannot get stats of a container.');
         },
+        async stop() {
+          if (ctrl.signal.aborted) {
+            // noop if already stopped
+            return;
+          }
+          await ctr.stop({ t: 0 });
+        },
+        async remove() {
+          if (ctrl.signal.aborted) {
+            // noop if already removed
+            return;
+          }
+          await ctr.remove({ force: true });
+        },
```

Comment:
This is unnecessary.. Containers [are started with the auto-remove option](https://github.com/ardatan/graphql-mesh/blob/93cfd7e8ccbce1f3e4d2b61a302a71b6bc987d90/e2e/utils/tenv.ts#L322) which removes the container after it stops.

```suggestion
```